app/controllers/donations.js

```diff
@@ -140,6 +140,10 @@ module.exports = (app) => {
                 }
               };
 
+              if (process.env.NODE_ENV === 'production') {
+                charge.application_fee_percent = constants.OC_FEE_PERCENT;
```

Comment:
well, but we need a different stripe account otherwise we have the issue `Can only apply an 'application_fee' when the request is made on behalf of another account (using an OAuth key or the Stripe-Account header)`